INSTALL

```diff
@@ -638,14 +638,13 @@ The specific libraries and their options are:
 - OCCA (optional), used when MFEM_USE_OCCA = YES.
   URL: https://libocca.org
   Options: OCCA_DIR, OCCA_OPT, OCCA_LIB.
-  Versions: OCCA >= 1.0.9.
+  Versions: OCCA >= 1.0.10.
```

Comment:
Technically, OCCA >= 1.0.10 is a requirent of libCEED 0.7 -- mfem can still use 1.0.9. Doesn't hurt to update it though. 😃